alcs-api/src/application/application.controller.ts

```diff
@@ -153,7 +157,18 @@ export class ApplicationController {
       highPriority: application.highPriority,
     });
 
-    const mappedApps = await this.applicationService.mapToDtos([app]);
+    if (app.assigneeUuid !== existingApplication.assigneeUuid) {
+      this.notificationService.createForApplication(
```

Comment:
Is await omitted on purpose?

Reply:
Yes, I figure if something fails with notifications we don't really need to alert the user or abort the return of the data below.